packages/installations/src/util/precondition-checks.test.ts

```diff
@@ -0,0 +1,99 @@
+/**
+ * @license
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { assert } from 'chai';
+import {
+  isValidAppIdFormat,
+  isValidApiKeyFormat
+} from '../util/precondition-checks';
+
+describe('precondition checks', () => {
+  it('returns true if App ID is valid', () => {
+    assert.isTrue(isValidAppIdFormat('1:123456789:android:abcdef'));
+    assert.isTrue(
```

Comment:
(For all other asserts) I think core folks want to have a consistence testing style by using behavior testing (i.e. expect). Please consider swapping

Reply:
done